docs/app/pages/components/components-sections/tables/sorting/snippets/app.html

```diff
@@ -21,10 +34,12 @@
         </ul>
 
         <!-- Sort Direction Toggle -->
-        <span class="sort-icon hpe-icon text-muted"
+        <button class="sort-icon hpe-icon text-muted"
+            aria-label="Toggle sort direction"
+            [attr.aria-sort]="descending ? 'descending' : 'ascending'"
```

Comment:
aria-sort only applies to table/grid header elements. The state of the button could be communicated via the aria-label.